File: BOJ/2675.cpp
#include<bits/stdc++.h>
using namespace std;

int T;

void proc(int R, string S) {
    for (int i = 0; i < S.length(); i++) {
        for (int j = 0; j < R; j++)
            cout << S[i];
	}
}

void input() {
    int R;
    string S;
    for (int testcase = 0; testcase < T; testcase++) {
        cin >> R >> S;
        proc(R, S);
        cout << "\n";
    }
}

int main() {
    cin >> T;
    input();
    
    return 0;
}

File: BOJ/2675.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
    
    public void input() throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        int T, R;
        String S;
        T = Integer.parseInt(br.readLine());
        for (int testcase = 0; testcase < T; testcase++) {
            String[] str = br.readLine().split(" ");
            R = Integer.parseInt(str[0]);
            S = str[1];
            proc(R, S);
            System.out.println();
        }
    }
    
    private void proc(int R, String S) {
        for (int i = 0; i < S.length(); i++) {
            for (int j = 0; j < R; j++)
                System.out.print(S.charAt(i));
        }
    }
    
    public static void main(String args[]) throws Exception {
        Main main = new Main();
        
        main.input();
    }
}
